client/src/lib/TinyMCEActionRegistrar.js

```diff
@@ -83,11 +84,13 @@ class TinyMCEActionRegistrar {
   getSortedActions(menu, configId, includingGlobal = true) {
     const actions = this.getActions(menu, configId, includingGlobal);
     return actions.sort((a, b) => {
-      if (a.priority !== b.priority) {
-        return a.priority < b.priority;
+      // Sort by descending priority
```

Comment:
`sort` expects a callback that will return a numeric value. Firefox is OK with a boolean but Chrome isn't.

Reply:
https://stackoverflow.com/questions/1969145/sorting-javascript-array-with-chrome